.prettierrc

```diff
@@ -1,6 +1,4 @@
 {
-  "singleQuote": true,
-  "trailingComma": "es5",
-  "requirePragma": false,
-  "arrowParens": "always"
-}
\ No newline at end of file
+  "tabWidth": 2,
```

Comment:
Since all other files have been formatted using previous config, it would be better to let the config remain as it is